src/data/faq.json

```
@@ -3659,7 +3693,7 @@
                 {
                     "term": "TAN",
                     "anchor": "tan",
-                    "description": "10-digit letter number combination for reporting positive test results. These can be requested via the TAN hotline. <b>National:</b> 0800 7540002 and <b>international:</b> +49 30 498 75402"
+                    "description": "<p>Updated ??????r.</p>10-digit letter number combination for reporting positive test results. These can be requested via the TAN hotline. <b>National:</b> 0800 7540002 and <b>international:</b> +49 30 498 75402. <p>Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of  ??????h.</p><a href='#warn_without_tan' target='_blank' >How do I warn without teleTAN?</a>"

```

Remove one ? and punctuation from ??????r, make it bold
Remove an unnecessary space before ??????h
```suggestion
                    "description": "<p><b>Updated ?????r</b></p>10-digit letter number combination for reporting positive test results. These can be requested via the TAN hotline. <b>National:</b> 0800 7540002 and <b>international:</b> +49 30 498 75402. <p>Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.</p><a href='#warn_without_tan' target='_blank' >How do I warn without teleTAN?</a>"
```